wiki/en/Installation-for-Linux.md

```diff
@@ -54,22 +54,43 @@ For installers on other distributions, see their package managers and [Repology]
 
 ## Set up your hardware
 
+If no instance of a JACK server is running, one is normally started automatically.
```

Comment:
```suggestion
If no instance of a JACK server is running, one typically starts automatically.
```

I'm not sure when this isn't automatic (the abnormal case). Is the manual case abnormal? Atypical? Special? Unusual?

Reply:
I think the best way to understand how it works is to test it and see what happens.
As that's not what this PR is about, I think we should concentrate on PipeWire, especially as this sentence hasn't been changed by my commits.

Reply:
I think hitting accept is easy and appropriate for any improvement in any PR. But if you don't believe I've improved anything, most certainly ignore me.

Reply:
From my point of view, opening an issue to submit a problem and if it is actually a real problem, submit a PR to solve the problem linking the issue to solve the problem is the best way to stay readable and understandable inside a multideveloppers project. That's why I am writting that.
I don't know how Jamulus behaves when there's no JACK server running. You are right.

Reply:
What I can say about JACK's clients behaviour is that when no server is running, the client is starting a server.
The server doesn't start automatically by itself.

Reply:
After 20 years in these settings, I've concluded that content production shouldn't operate like code production. Our users need readable and understandable information and I will keep trying to deliver it with your help.

Reply:
When Jamulus client starts a new instance of JACK server, it typically runs just fine? I haven't tested any of this, as you suggested. But would you consider:

"If no instance of JACK server is running when Jamulus Client starts, it will start a new instance and connect to it. A misconfiguration or error might prevent the JACK server from starting."

This separates the happy path from the error scenario. Do you think _misconfiguration and error_ cover the failure scenarios you're aware of? Is it always a misconfiguration? Perhaps lack of RAM? How common are failure scenarios?

Reply:
> When Jamulus client starts a new instance of JACK server, it typically runs just fine?

Yes.
> "If no instance of JACK server is running when Jamulus Client starts, it will start a new instance and connect to it. A misconfiguration or error might prevent the JACK server from starting."
> 
> This separates the happy path from the error scenario. Do you think _misconfiguration and error_ cover the failure scenarios you're aware of?

No, it is more complex than that. And that's why the preferred method is to start a JACK server before running Jamulus client.
> Is it always a misconfiguration? Perhaps lack of RAM? How common are failure scenarios?

It is not always a misconfiguration, it can be the unplugged audio interface, the audio card already used by another software ... for example. Not a lack of RAM. I don't know how common are failure scenarios.

Reply:
Changing everything with the two last commits. Should outdate this discussion.

Reply:
Resolving.